### PR TITLE
faust: Add formula

### PR DESCRIPTION
**Pull request by thiblahute**

### CODE REVIEW

File: Library/Formula/faust.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  url "https://downloads.sourceforge.net/project/faudiostream/faust-0.9.73.tgz"
+  sha256 "6512828b00e7a5a6d5ed6efc44dc54381991275abdc8aacf3a5deecd67675a74"
+
+  def install
+    system "make", "PREFIX=#{prefix}"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "which", "faust"
```
Comment (dunn) [inline Library/Formula/faust.rb]: We'll need a better test; something that checks the functionality of the installed software.

**Comment by dunn** [comment]
There's an audit failure too:
```
==> audit problems
faust:
 * The homepage is not reachable (curl exit code 28)
```
It's working for me now, so maybe that was just a fluke.  If you push again with a new test it'll re-run.

File: Library/Formula/faust.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/faust.rb]: Ping on Alex's comment?

**Comment by thiblahute** [comment]
I updated the patch.

File: Library/Formula/faust.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  url "https://downloads.sourceforge.net/project/faudiostream/faust-0.9.73.tgz"
+  sha256 "6512828b00e7a5a6d5ed6efc44dc54381991275abdc8aacf3a5deecd67675a74"
+
+  def install
+    system "make", "PREFIX=#{prefix}"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/faust", "--version"
```
Comment (DomT4) [inline Library/Formula/faust.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Looks like there's a bunch of examples in this upstream PDF: http://faust.grame.fr/images/faust-tutorial2.pdf

**Comment by bfontaine** [comment]
Ping re Dom’s comment?

**Comment by thiblahute** [comment]
Sorry I do not have a mac right now, I will try to think about doing that when I get one back, or please ping me again later.

**Comment by mikemcquaid** [comment]
Ok, happy to reopen when you get access to one again.

**Pull request closed without merging**